### ISSUE TITLE
Rename this repository to phparsenal as GH organization page

### ISSUE THREAD

**Issue by nochso** [phparsenal/roadmap#2]
According to https://pages.github.com

A repository with the same name as the organization would let us
`git clone https://github.com/phparsenal/phparsenal.github.io`

Push HTML to that repository and it will be served from our new website:
http://phparsenal.github.io

**Comment by samayo** [comment]
didn't know the name was already accepted. Good idea btw

**Comment by nochso** [comment]
Not really accepted, but considering the low activity we just needed something to get started.

**Comment by samayo** [comment]
Yeah, good idea. I won't be anywhere near my pc this week, but I'll catch up soon after. It would be nice if someone can open issues/tasks and assign them to each individual in the group .. that way ... it would help to come and quickly do your part, if things like time and full dedication to go through the discussion become an issue.
